airbrake.gemspec

```diff
@@ -40,6 +40,8 @@ DESC
   s.add_development_dependency 'appraisal', '~> 2'
   s.add_development_dependency 'rack', '~> 1'
   s.add_development_dependency 'webmock', '~> 2'
+  s.add_development_dependency 'sneakers', '~> 2'
+  s.add_development_dependency 'rr'
```

Comment:
Why can't we use RSpec mocks (we use them for the rest of this project)?

Reply:
Yes, we can. :)
I just copied some test setup from the sneakers gem and was lazy, but now I have rewritten it to use RSpec mocks like you suggested.